eng/pipelines/templates/stages/archetype-net-release.yml

```diff
@@ -80,6 +70,14 @@ stages:
                         PackageName: ${{artifact.name}}
                         ServiceName: ${{parameters.ServiceDirectory}}
                         ForRelease: true
+                    - task: PowerShell@2
+                      inputs:
```

Comment:
You will also want to add `pwsh: true` so that it uses powershell core instead of windows powershell.